Gui/opensim/view/src/org/opensim/view/pub/OpenSimDB.java

```diff
@@ -81,7 +81,7 @@
      */
     static public CloseModelDefaultAction getCurrentCloseModelDefaultAction() {
         String defaultCloseActionString="Prompt";
-        String saved = Preferences.userNodeForPackage(TheApp.class).get("DefaultCloseAction", defaultCloseActionString);
+        String saved = TheApp.getCurrentVersionPreferences().get("DefaultCloseAction", defaultCloseActionString);
```

Comment:
I can't tell whether this preference is user-facing or not. If not, presumably it should be "Internal.DefaultCloseAction"; if so, "Default Close Action"?